sphinx/source/docs/user_guide/export.rst

```diff
@@ -68,35 +65,33 @@ saving the image to a file, you can use the lower-level function
 
 .. _userguide_export_svg:
 
-SVG Generation
+SVG generation
 --------------
 
-Bokeh also supports replacing the HTML5 Canvas plot output with an SVG element
-that can be edited in image editing programs such as Adobe Illustrator and/or
-converted to PDFs.
+Bokeh can also replace the HTML5 Canvas plot output with a Scalable Vector
+Graphics (SVG) element that can be edited in image editing programs such
+as Adobe Illustrator and/or converted to a PDF.
 
 The SVG output isn't as performant as the default Canvas backend when it comes
 to rendering a large number of glyphs or handling lots of user interactions like
 panning.
 
-Like PNGs, in order to create an SVG with a transparent background, users
-should set the ``Plot.background_fill_color`` and ``Plot.border_fill_color``
-properties to ``None``.
+To create an SVG with a transparent background, set the ``Plot.background_fill_color``
+and ``Plot.border_fill_color`` properties to ``None``, same as with PNG output.
 
 Limitations
 ~~~~~~~~~~~
 
-It's not possible to create a single SVG for a layout of plots, as each plot
-is its own distinct SVG element. Alternatively, it's possible to download an
-SVG plot using a SaveTool from the toolbar. However, currently an SVG export
-of a plot with a toolbar will have a blank area where the toolbar was rendered
-in the browser.
+You can't create a single SVG for a layout of plots because each plot will
+produce its own distinct SVG element. You can, however, download an SVG plot
+using a SaveTool from the toolbar. Note that in this case the exported file
+will have a blank area where the toolbar used to be.
```

Comment:
I think this entire para needs more attention. These two cases don't really have anything to do with one another, but it reads (since the original) like somehow the savetool is a solution to the first issue with layouts. 

Suggest that the last two sentences be combined:

> You can download an SVG plot using a SaveTool from the toolbar (note that the exported file
will have a blank area where the toolbar was).

And I think this should be separated from the first sentence as well, either two very short one-sentence para's or turn in to a bulleted list maybe?

@tcmetzger about this:

> You can't create a single SVG for a layout of plots because each plot will produce its own distinct SVG element.

I wanted to see if you have any suggestion for different wording. This is a limitation of the library, but in this context the "You can't ..." reads to me like "You aren't allowed" TBH I prefer direct statements like "It is not possible to..." in cases where things are.... not possible. What is the compromise update here?

Reply:
Oh I see, that original "Alternatively,..." looks to be copy-pasta from another section. It was definitely badly worded from the start.

Reply:
Well, "it is not possible" sounds pretty good to me! Or you could turn around the sentence a bit: "Each plot is its own SVG element. Therefore, it is not possible to automatically create a single SVG combining all plots in a gridplot layout."